src/WebRTC/RELEASE_NOTES.md

```diff
@@ -1,3 +1,8 @@
+### 1.2.1
```

Comment:
This adds new features and has some small breaking changes (the type changes), maybe at least the minor version should be bumped (1.3.0)?